src/WebexSDKAdapter.js

```diff
@@ -12,5 +12,23 @@ export default class WebexSDKAdapter {
   constructor(sdk) {
     this.peopleAdapter = new PeopleSDKAdapter(sdk);
     this.roomsAdapter = new RoomsSDKAdapter(sdk);
+    this.sdk = sdk;
+  }
+
+  /**
+   * Call to register the device to the cloud and start the
```

Comment:
I think we should make this description a little more abstract, in case we do more things with method in the future.
Here's my -more generic- suggestion:
```
Connect to Webex services so SDK can listen for data updates.